`test_wheel.WheelTestCase.test_abi` fails on freethreading CPython versions

**Describe the bug**
When running the test suite using a freethreaded build of CPython, the following test fails:

```pytb
======================================================================
FAIL: test_abi (test_wheel.WheelTestCase.test_abi)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/distlib/tests/test_wheel.py", line 726, in test_abi
    self.assertEqual(''.join(parts), abi)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
AssertionError: 'cp313' != 'cp313t'
- cp313
+ cp313t
?      +
```

**To Reproduce**
```
uv venv -p 3.13t
uv pip install -e .
. .venv/bin/activate
python tests/test_all.py
```

**Expected behavior**
Tests passing.

**Environment**
 - OS, Gentoo Linux amd64
 - Version of this library: 32301789a7815de0e74f57fe013ae52af717a3da

**Additional information**
I'll submit a pull request.

--- a/.github/workflows/package-tests.yml
+++ b/.github/workflows/package-tests.yml
@@ -28,7 +28,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12', '3.13', 'pypy-2.7', 'pypy-3.9']
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12', '3.13', '3.13t', 'pypy-2.7', 'pypy-3.9']
         exclude:
           # macos-latest does not have Python 3.7
           - os: macos-latest

--- a/.github/workflows/package-tests.yml
+++ b/.github/workflows/package-tests.yml
@@ -28,7 +28,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12', '3.13', '3.13t', 'pypy-2.7', 'pypy-3.9']
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12', '3.13', '3.13t', 'pypy-2.7', 'pypy-3.9', 'pypy-3.10', 'pypy-3.11']
         exclude:
           # macos-latest does not have Python 3.7
           - os: macos-latest

--- a/distlib/wheel.py
+++ b/distlib/wheel.py
@@ -71,6 +71,8 @@ else:
                     us = sysconfig.get_config_var('Py_UNICODE_SIZE')
                     if us == 4 or (us is None and sys.maxunicode == 0x10FFFF):
                         parts.append('u')
+            if bool(sysconfig.get_config_var("Py_GIL_DISABLED")):
+                parts.append('t')
         return ''.join(parts)
 
     ABI = _derive_abi()

--- a/tests/test_wheel.py
+++ b/tests/test_wheel.py
@@ -719,6 +719,8 @@ class WheelTestCase(DistlibTestCase):
                 us = sysconfig.get_config_var('Py_UNICODE_SIZE')
                 if us == 4 or (us is None and sys.maxunicode == 0x10FFFF):
                     parts.append('u')
+            if bool(sysconfig.get_config_var("Py_GIL_DISABLED")):
+                parts.append('t')
         if vi < (3, 5):
             abi = ABI
         else:

